css/style.css

```diff
@@ -1570,6 +1587,9 @@ iframe.campaign-alert {
     background: #1abc9c;
     text-shadow: none;
     border: none;
+    width: 20%;
+    height: 20%;
+    margin: 0% 40%;
```

Comment:
You can't change this class because it affects other pages (take a look at how it messed up the learn more buttons in the page enrollment). If you need to add specific css, create an additional class, say ```volunteer-buttons```, and add the tag to the button list of classes.

Reply:
also, can you make the button a bit bigger?